CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to Anyledger wallet
```

Comment:
This whole file looks like a copy-paste from another project from briandk. It even has a few parts written in first person. Are you sure we have the right to derivate from that document? What is the license of the original file? Did you ask the original author if it was OK to use this as guidelines in other projects? Are you sure this applies to us? The license seems wrong, and other things seem a bit off...

Reply:
>This whole file looks like a copy-paste from another project from briandk

... which is based on copy-paste of facebook's guidelines.

Reply:
still, there are reference in first person, so this look pretty amateuristic

Reply:
I agree, let's reword in third person.